lib/bindings/kvbm/python/kvbm/vllm_integration/consolidator_config.py

```
@@ -9,23 +9,17 @@
 
+from kvbm.utils import get_consolidator_mode, is_truthy
 from vllm.distributed.kv_events import ZmqEventPublisher
 
-logger = logging.getLogger(__name__)
-
+__all__ = [
+    "get_consolidator_mode",
+    "is_truthy",
+    "should_enable_consolidator",
+    "get_consolidator_endpoints",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)